praw/objector.py

```diff
@@ -21,18 +21,14 @@ def parse_error(cls, data):
             # assume it's a possibility here.
             return None
 
-        errors = data.get("json", {}).get("errors")
-        if errors is None:
+        errors = data.get("json", {}).get("errors", ())
+        if not errors:
```

Comment:
Are there any valid cases now where errors is set but the list is empty? If not I propose we keep this conditional as it was previously.

Reply:
It’s actually fairly common. A quick grep of the cassettes reveal that a majority of requests are like this.

```
$ gfind tests/integration/cassettes/ -name '*.json' | gwc -l
492
$ ggrep -FIr '\"errors\": []' tests/integration/cassettes/ | gwc -l
335
```